### PR TITLE
ticdc: fix alert doc

### PR DESCRIPTION
**Pull request by CharlesCheung96**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Ref: https://github.com/pingcap/docs-cn/pull/19221

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.5 (TiDB 8.5 versions)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/19221
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by hfxsd — APPROVED** [review]

File: ticdc/ticdc-alert-rules.md [hunk 1]
```diff
@@ -86,61 +72,48 @@ Warning alerts are a reminder for an issue or error.
 
     Collect TiCDC logs to locate the root cause.
 
-### `cdc_sink_flush_duration_time_more_than_10s`
+### `cdc_no_owner`
 
 - Alert rule:
-
-    `histogram_quantile(0.9, rate(ticdc_sink_txn_worker_flush_duration[1m])) > 10`
+
```
Comment (qiancai) [inline ticdc/ticdc-alert-rules.md:78]:
```suggestion

```

File: ticdc/ticdc-alert-rules.md [hunk 2]
```diff
@@ -86,61 +72,48 @@ Warning alerts are a reminder for an issue or error.
...
 - Alert rule:
-
-    `histogram_quantile(0.9, rate(ticdc_sink_txn_worker_flush_duration[1m])) > 10`
+  
+    `sum(rate(ticdc_owner_ownership_counter[240s])) < 0.5`
 
 - Description:
-
-    It takes a replication task more than 10 seconds to write data to the downstream database.
+
```
Comment (qiancai) [inline ticdc/ticdc-alert-rules.md:82]:
```suggestion

```

File: ticdc/ticdc-alert-rules.md [hunk 3]
```diff
@@ -86,61 +72,48 @@ Warning alerts are a reminder for an issue or error.
...
-- Alert rule:
-
-    `changes(ticdc_processor_checkpoint_ts[1m]) < 1`
-
-- Description:
-
-    A replication task has not advanced for more than 1 minute.
-
-- Solution:
+    Collect TiCDC logs to locate the root cause.
```
Comment (qiancai) [inline ticdc/ticdc-alert-rules.md:87]:
```suggestion
    Collect TiCDC logs to identify the root cause.
```

File: ticdc/ticdc-alert-rules.md [hunk 4]
```diff
@@ -86,61 +72,48 @@ Warning alerts are a reminder for an issue or error.
...
 
 - Alert rule:
 
-    `histogram_quantile(0.9, rate(ticdc_puller_entry_sorter_sort_bucket{}[1m])) > 1`
+    `(max_over_time(ticdc_owner_status[1m]) == 1 or max_over_time(ticdc_owner_status[1m]) == 6) > 0`
 
 - Description:
-
-    The delay of TiCDC puller entry sorter is too high.
+
```
Comment (qiancai) [inline ticdc/ticdc-alert-rules.md:97]:
```suggestion

```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: ticdc/ticdc-alert-rules.md [hunk 5]
```diff
@@ -86,61 +72,45 @@ Warning alerts are a reminder for an issue or error.
...
-
-    `changes(ticdc_processor_checkpoint_ts[1m]) < 1`
-
-- Description:
-
-    A replication task has not advanced for more than 1 minute.
-
-- Solution:
-
-    See [TiCDC Handles Replication Interruption](/ticdc/troubleshoot-ticdc.md#how-do-i-handle-replication-interruptions).
```
Comment (hfxsd) [inline ticdc/ticdc-alert-rules.md:87]:
```suggestion
```

**Pull request merged**